content/admin/onboard_team/CreateUsers.md

```diff
@@ -4,41 +4,108 @@ description: "Creating Users in Contrast TeamServer"
+* Check the box if you want to **Require email activation** instead of requiring a password.
+* Choose which of the **System Roles** should apply to the user in the dropdown menu. The default is **None**. 
+* Choose the **Organization** to which the user belongs. 
+* Once you decide on an organization, you can choose the default **Organization Role** in the dropdown menu as well as an **Application Access Group** in the multiselect field.
+* Choose the **Date Format**, **Time Format** and **Time Zone** in the dropdown fields. 
+* The box to **Use Organization Settings** is checked by default. You can uncheck the box to create your own settings using the **Access** toggle or the box to **make user API only**. 
+* Click the **Add** button to save the user's information. 
 
->**Note:** Make sure you verify the [Role](admin-manageorgsroleperm.html#roles) so that users will have the privileges to carry out the actions you intend.
+<a href="assets/images/User-system-admin.png" rel="lightbox" title="Add a user as a Super Administrator"><img class="thumbnail" src="assets/images/User-system-admin.png"/></a>
 
-<a href="assets/images/Create_User.png" rel="lightbox" title="Add User"><img class="thumbnail" src="assets/images/Create_User.png"/></a>
 
-Users will receive an activation email to join Contrast. You will be able to check their status from the main Users table to identify who is waiting activation, active/inactive, or is locked out of their account based on security policy.
+### Multiple users 
+
+To bulk add users, click the upload icon above the grid to import a spreadsheet with the users' information. The spreadsheet must be CSV format, and include the following fields for each user. **All field headings and values in the spreadsheet must be formatted as shown.** 
+
+* First Name 
+* Last Name 
+* Email 
+* Organization 
+* Organization Role <br> (**View**, **Edit**, **Rules_admin**, or **Admin**)
+
+> **Note:** The **Email** field appears as **Username** for users who have [HTTP Header](installation-setupauth.html#http-proxy) or [LDAP authentication](installation-setupauth.html) configured. 
+
+You may also include the following information for each user in the spreadsheet: 
+
+* Email Activation <br> (If "None", default is **Required Password**)
+* System Administration <br> (Default is **Off**)
+* Application access group <br> (View, Edit, Rules Admin, Admin or custom group names; format multiple group names as **GroupA&GroupB&GroupC**)
+* Date Format <br> (Default is the organization setting, such as **MM/dd/YYYY**)
+* Time Format <br> (Default is the organization setting, such as **hh:mm a**)
+* Time zone <br> (Default is the organization time zone)
+* Access <br> (Default is **On**)
+* API only <br> (Default is **Off**)
+
+> **Note:** Hover over the upload icon for a reminder of required file formats and information fields. You can also download a **CSV template** by clicking the link in the tooltip. 
```

Comment:
@kaitlynmortimer We can remove the part about "required file formats" (since we're only doing CSV now) and just say "...for a reminder of required information fields." Also, @ScoRobi is supposed to provide you with the actual templates to link to in the article. There will be four - two each for Org and SuperAdmin.